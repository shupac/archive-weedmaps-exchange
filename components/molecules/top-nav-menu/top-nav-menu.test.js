import React from 'react';
import { shallow } from 'enzyme';
import Menu from './';

describe('Top Nav Menu', () => {
  it('Menu', () => {
    const tree = shallow(<Menu active="deals" />);
    expect(tree).toMatchSnapshot();
  });
});
