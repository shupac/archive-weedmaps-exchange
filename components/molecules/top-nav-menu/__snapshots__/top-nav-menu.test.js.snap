// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Nav Menu Menu 1`] = `
<top-nav-menu__MenuContainer>
  <top-nav-menu__IconWrapper>
    <WithTheme(Menu)
      fill="#999999"
      size={
        Object {
          "height": "16px",
          "width": "24px",
        }
      }
    />
  </top-nav-menu__IconWrapper>
</top-nav-menu__MenuContainer>
`;
